services/oada_upload/dev-startup.sh

```diff
@@ -0,0 +1,8 @@
+#! /bin/bash
+
+echo "-----------------------------------------------------"
+echo "build: "
+yarn run build
```

Comment:
Wny building in a container at runtime? Should be built in the Dockerfile if a build stage is needed.

Reply:
Good point, I was just trying to navigate TS nonsense and finally got things working by building and then running rather than using ts-node which is what was originally there.  ts-node has an issue with declaration files that just doesn't work at all.  It would make sense now that things are working to move the build step to the Dockerfile rather than the entrypoint, although when developing and mapping your local code into the container, it is better to just build in the entrypoint so you can just restart the service on code changes.